cnho_stables/assetlist.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cnho stables",
```

Comment:
do not use spaces or _ in chainname property. Just "cnhostables" is fine

Reply:
or even "cnho"

Reply:
ok ,use the cnhostables to replace it